setup.py

```diff
@@ -99,12 +99,15 @@
     ],
     extras_require={
         'test': [
+            'brasil.gov.agenda',
```

Comment:
none of these inclusions are needed: all of those packages are already listed in `install_requires`